versions/1/spec.yml

```diff
@@ -17,11 +17,11 @@ spec:
   name: "changelog.yml"
   required: true
   $ref: "./changelog.spec.yml"
-- description: Folder containing dataset definitions
+- description: Folder containing data stream definitions
   type: folder
-  name: dataset
+  name: data_stream
```

Comment:
While we're at it, should we rename this folder to `data_streams` (plural), since it typically contains multiple sub-folders, one for each data stream?

Reply:
I personally like that it is singular because each file path is a single data stream: `data_stream/access/`. But I know quite a few people disagree and a discussion me and @exekias have for years 😆  So I'm good with both.

Reply:
I vote for the plural, similarly to REST naming conversion:

1. Document is singular: account, customer
2. Set of documents is plural: accounts, customers

so: data_streams

Reply:
btw. do we know the scope of changes caused by this single update?

Reply:
> btw. do we know the scope of changes caused by this single update?

Once https://github.com/elastic/package-spec/pull/14 is merged (we're pretty close on that one I think), we can run `elastic-package lint` with this PR here and get an idea of the changes that will be needed across all the packages.

Reply:
Looking at some other existing package folders that represent collections of things, we don't have any consistency ☹️ . Some collection folders use singular names, e.g., `kibana/dashboard/`, `kibana/visualizations/`, and `img/`. And some collection folders use plural names, e.g. `docs/` and `dataset/<foo>/fields`. 

I don't have strong opinions in either direction but I slightly favor plural names for folders that represent collections of things. I do have a strong opinion about trying to be consistent 🙂.

Reply:
@ycombinator There is something indeed odd with `visualizations` vs `datashboard`. I thought we took the name of the saved object here but it seems `visualization` is singular in KB :-( ++ on consistency.

Reply:
I checked some packages and couldn't find visualizations? Where did you spot that?

Reply:
Sorry @ruflin, that was a typo in my comment above. I meant `kibana/visualization` not `kibana/visualizations`. Edited and fixed now.